### ISSUE TITLE
Code generator doesn't work with "typescript-apollo-angular"

### ISSUE THREAD

**Issue by kover** [dotansimha/graphql-code-generator#5065]
**Describe the bug**

Facing with the following error when trying to generate files for the Angular project:

```
> graphql-codegen --config codegen.yml

  ✔ Parse configuration
  ❯ Generate outputs
    ❯ Generate src/app/generated/graphql.ts
      ✔ Load GraphQL schemas
      ✔ Load GraphQL documents
      ✖ Generate
        → Unable to find template plugin matching typescript-apollo-angular
    ✔ Generate ./graphql.schema.json


 Found 1 error

  ✖ src/app/generated/graphql.ts
    Unable to find template plugin matching 'typescript-apollo-angular'
    Install one of the following packages:


    - @graphql-codegen/typescript-apollo-angular
    - @graphql-codegen/typescript-apollo-angular-template
    - @graphql-codegen/typescript-apollo-angular-plugin
    - graphql-codegen-typescript-apollo-angular
    - graphql-codegen-typescript-apollo-angular-template
    - graphql-codegen-typescript-apollo-angular-plugin
    - codegen-typescript-apollo-angular
    - codegen-typescript-apollo-angular-template
    - typescript-apollo-angular
    Error: Unable to find template plugin matching typescript-apollo-angular
        at getPluginByName (/Users/macbookuser/Developer/webui-2.0/node_modules/@graphql-codegen/cli/bin.js:328:11)
    Error: Unable to find template plugin matching typescript-apollo-angular
        at getPluginByName (/Users/macbookuser/Developer/webui-2.0/node_modules/@graphql-codegen/cli/bin.js:328:11)


Running lifecycle hook "afterStart" scripts...
[CLI] Loading Schemas
[CLI] Loading Schemas
[CLI] Loading Documents
[CLI] Loading Documents
[CLI] Generating output
[CLI] Generating output
[CLI] Exited with an error DetailedError: Unable to find template plugin matching typescript-apollo-angular
    at getPluginByName (/Users/macbookuser/Developer/webui-2.0/node_modules/@graphql-codegen/cli/bin.js:328:11) {
  details: '\n' +
    "        Unable to find template plugin matching 'typescript-apollo-angular'\n" +
    '        Install one of the following packages:\n' +
    '        \n' +
    '        \n' +
    '        - @graphql-codegen/typescript-apollo-angular\n' +
    '        - @graphql-codegen/typescript-apollo-angular-template\n' +
    '        - @graphql-codegen/typescript-apollo-angular-plugin\n' +
    '        - graphql-codegen-typescript-apollo-angular\n' +
    '        - graphql-codegen-typescript-apollo-angular-template\n' +
    '        - graphql-codegen-typescript-apollo-angular-plugin\n' +
    '        - codegen-typescript-apollo-angular\n' +
    '        - codegen-typescript-apollo-angular-template\n' +
    '        - typescript-apollo-angular\n' +
    '      ',
  source: 'src/app/generated/graphql.ts',
  context: [Object: null prototype] {}
}
Something went wrong
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @companyname/webui@2.0.0 gql-generate: `graphql-codegen --config codegen.yml`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @companyname/webui@2.0.0 gql-generate script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
```

**To Reproduce**

<!--
#############
#############
🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 

PLEASE READ THE FOLLOWING: 
You can use this template to create a live sandbox of the issue: 
https://codesandbox.io/s/github/dotansimha/graphql-code-generator-issue-sandbox-template
Make sure sure fork this template, and run `yarn generate` in the terminal. 
Please make sure the Codegen and plugins version under `package.json` matches yours. 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~ ISSUES WITH A LIVE REPRODUCTION WILL BE ANSWERED QUICKLY ~~~~~~~
~~~~~~ ISSUES WITHOUT REPRODUCTION MIGHT GET CLOSED WITHOUT RESPONSE ~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#############
#############
-->

1. My GraphQL schema:

```graphql
# Put your schema here
```

2. My GraphQL operations:

```graphql
# Put your operations here
```

3. My `codegen.yml` config file:

```yml
overwrite: true
schema: "schema.js"
documents: "src/**/*.graphql"
config:
  declarationKind: "interface"
generates:
  src/app/generated/graphql.ts:
    plugins:
      - "typescript"
      - "typescript-operations"
      - "typescript-apollo-angular"
  ./graphql.schema.json:
    plugins:
      - "introspection"
```

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->

Generated output.

**Environment:**

- OS: macOS Catalina 10.15.7
- `@graphql-codegen/cli`: `1.19.1`
- `@graphql-codegen/fragment-matcher`: `2.0.0`
- `@graphql-codegen/introspection`: `1.18.0`
- `@graphql-codegen/typescript`: `1.17.11`
- `@graphql-codegen/typescript-apollo-angular`: `2.1.0`
- `@graphql-codegen/typescript-apollo-client-helpers`: `^1.1.0`
- `@graphql-codegen/typescript-operations`: `1.17.9`
- NodeJS: 13.13.0

**Additional context**
<!-- Add any other context about the problem here. -->

**Comment by dotansimha** [comment]
Please share a live reproduction.

**Comment by kover** [comment]
for me, it happens after I'm using the update process in Angular. For example, I've tried 

`ng update angular-apollo @graphql-codegen/cli @graphql-codegen/introspection @graphql-codegen/typescript @graphql-codegen/typescript-apollo-angular @graphql-codegen/typescript-operations`

and also 

`ng update apollo-angular && npm uninstall @graphql-codegen/introspection @graphql-codegen/typescript @graphql-codegen/typescript-apollo-angular @graphql-codegen/typescript-operations && npm install @graphql-codegen/introspection @graphql-codegen/typescript @graphql-codegen/typescript-apollo-angular @graphql-codegen/typescript-operations`

both ways show me an error on `graphql-codegen --config codegen.yml`. But I've found a workaround:
1. `ng update angular-apollo`
2. `ng update @graphql-codegen/cli`
3. `ng update @graphql-codegen/introspection @graphql-codegen/typescript @graphql-codegen/typescript-apollo-angular @graphql-codegen/typescript-operations`

After this, I see no errors.
Checked both ways twice and have no idea about the reason.
Also, updated to Angular 11, but I'm not sure that this helped.
I think the issue can be closed, as the search shows that no one else faced with such an issue.
